### ISSUE TITLE
Unable to find struct with explicit macro use on 2018 edition

### ISSUE THREAD

**Issue by Redrield** [intellij-rust/intellij-rust#3098]
## Environment

* **Intellij-Rust plugin version:** 0.2.0.2110-182
* **Rust toolchain version:** rustc 1.32.0-nightly
* **IDE name and version:** IntelliJ IDEA Ultimate 2018.2.6
* **Operating system:** Arch Linux, Linux 4.19.4-arch1-1-ARCH

## Problem description
On Rust 2018 edition, when trying to use a struct generated by the `bitflags` crate, the plugin recognizes the struct when I have an explicit `extern crate` marked with `#[macro_use]` in the root of the project, however fails when I remove that and use the `bitflags::bitflags` directly. 

## Steps to reproduce
Generate a simple struct with bitflags
```rust
bitflags! {
    struct Foo: u32 {
        const A = 0b00000000;
        const B = 0b10000000;
    }
}
```

Try to reference it in another struct, when using `extern crate`, and when `use`-ing the macro directly. The latter will result in an unresolved reference, despite being able to compile.

My entire codebase is at https://gitlab.com/Redrield/ds-rs. The problematic bitflag struct is under inbound::udp::types, the problematic reference is under inbound::udp.

**Comment by vlad20012** [comment]
Yep, it's a part of Rust 1.30 release which is not implemented yet. #2986
